packages/lambda-tiler/src/router.ts

```diff
@@ -0,0 +1,56 @@
+import { Const } from '@basemaps/shared';
```

Comment:
Imported directly from `@basemaps/lambda` as it was not pushed across to `@linzjs/lambda`